testdata/baselines/reference/submoduleAccepted/compiler/jsdocParamTagInvalid.types.diff

```diff
@@ -0,0 +1,10 @@
+--- old.jsdocParamTagInvalid.types
++++ new.jsdocParamTagInvalid.types
+@@= skipped -2, +2 lines =@@
+ === /a.js ===
+ /** @param {string} colour */
+ function f(color) {}
+->f : (color: any) => void
+->color : any
++>f : (color: string) => void
++>color : string
```

Comment:
Perhaps this is like the other comment, but why is this ending up succeeding? I'm confused how fixing a missing error (in another PR) would cause this to flip back to `any`.